src/shell/components/global-account/GlobalAccount.js

```diff
@@ -16,32 +17,30 @@ export default connect((state) => {
   };
 })(function GlobalAccount(props) {
   const [open, setOpen] = useState(false);
-  const ref = useRef();
+  const ref = useRef(null);
+
+  const handleClickOutside = (event) => {
```

Comment:
This function definition needs to stay inside the `useEffect.` As the reference will change on re-renders causing the `removeEventListener` to use a different reference, resulting in a memory leak.